__tests__/addresses.test.js

```diff
@@ -0,0 +1,11 @@
+import TestUtils from "./test-utils";
+
+describe("addresses api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .get("/wallet/addresses")
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.addresses.length).toBeGreaterThan(20);
```

Comment:
I feel the addresses are ordered by their indexes, right? In this case, we should check the first 5 address just to be sure.

Reply:
Done